### ISSUE TITLE
Fix Storybook Accessibility Violations 

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#282]
**User Story:**
As Spark we want to fix all storybook accessibility violations so that we can make fixes prior to go live of NDS.

**Notes:**
- This came up while testing NDS
- Research Issue sparkdesignsystem/spark-design-system#2444 

**AC:**
- I will no longer see Storybook Accessibility violations.

**Comment by ssabo1** [comment]
This has been broken down to component level Issues.
